charts/provider/templates/secret.yaml

```
@@ -26,6 +26,13 @@ endpoints:
     node-urls:
     {{- range $node := $interfaceEntry.nodes }}
       - url: {{ $node.endpoint | quote }}
+      {{- if $node.authConfig.enabled }}
+        auth-config:
+        {{- if $node.authConfig.authHeaders.enabled }}
+          auth-headers:
+            AUTH-X-HEADER: {{ $node.authConfig.authHeaders.authXHeader }}
```

we need AUTH-X-HEADER to be also configurable. 

so we can just do this:
```suggestion
            {{ $node.authConfig.authHeaders.authXHeader }}
```

---

Done